### PR TITLE
qa/tests: removed nautilus suites as EOL

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by batrick — APPROVED** [review]

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -55,27 +54,6 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
 
 ##########################
 
-#********** nautilus branch START - weekly
-
-25 13 * * 5  CEPH_BRANCH=nautilus; MACHINE_NAME=gibba; SUITE_NAME=kcephfs;  KERNEL=distro; /home/teuthology/bin/cron_wrapper /home/teuthology/bin/schedule_subset.sh 2999 $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL $KERNEL
-15 05 * * 0  CEPH_BRANCH=nautilus; MACHINE_NAME=gibba; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 100 -m $MACHINE_NAME -s krbd -k testing -e $CEPH_QA_EMAIL
```
Comment (idryomov) [inline qa/crontab/teuthology-cronjobs:61]: As I said in https://github.com/ceph/ceph/pull/41226#pullrequestreview-654736621, I would like to keep the weekly `krbd` run for now.

**Review by idryomov — CHANGES_REQUESTED** [review]

File: qa/crontab/teuthology-cronjobs (hunk 1 above)
Comment (yuriw) [reply qa/crontab/teuthology-cronjobs:61]: I thought we discussed it before `nautilus` EOL

**Comment by idryomov** [comment]
Nit: why are you moving the nautilus section after pacific?  Just remove everything except `krbd` and `kcephfs` lines and keep the nautilus section where it is (i.e. the diff should be just `-`s, no `+`s -- and just a single commit).

File: qa/crontab/teuthology-cronjobs [hunk 2]
```diff
@@ -106,28 +91,22 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
...
-
-20 07 * * 6   CEPH_BRANCH=pacific; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 100 -m $MACHINE_NAME -s upgrade-clients/client-upgrade-nautilus-pacific -k distro -e $CEPH_QA_EMAIL --suite-branch nautilus -p 500
-
-
-22 14 * * 6    CEPH_BRANCH=pacific; MACHINE_NAME=smithi; SUITE_NAME=upgrade:octopus-x; KERNEL=distro; /home/teuthology/bin/cron_wrapper /home/teuthology/bin/schedule_subset.sh 10 $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL $KERNEL -p 500 --force-priority
-
-25 01 * * 7  CEPH_BRANCH=pacific; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 100 -m $MACHINE_NAME -s upgrade/pacific-p2p -k distro -e $CEPH_QA_EMAIL
-
+31 03 * * 1,3,5,7   CEPH_BRANCH=pacific; MACHINE_NAME=smithi; SUITE_NAME=rados;      KERNEL=distro; /home/teuthology/bin/cron_wrapper /home/teuthology/bin/schedule_subset.sh 99999 $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL $KERNEL -p 500 --force-priority
```
Comment (idryomov) [inline qa/crontab/teuthology-cronjobs:94]: Now there seem to be some seemingly unrelated changes such as `31 03 * * 0` -> `31 03 * * 1,3,5,7`.  Not saying they are wrong but there is no explanation for these adjustments.

**Comment by idryomov** [comment]
I have only just noticed that the kcephfs suite seems to be scheduled with the distro kernel.  It's not going to be of any use here, as the intent is to make sure that the "testing" kernel continues to work with nautilus.  Please drop it and leave just the krbd suite.

Otherwise looks good but please squash all those commits into a single commit.

**Review by idryomov — APPROVED** [review]
